### PR TITLE
Custom buttons in FeatureInfoSection

### PR DESCRIPTION
**Pull request by rdengate**
Can now add extra components to FeatureInfoSection. Also, renamed Download Data to Download Data for this Feature.

### CODE REVIEW

File: lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx [hunk 1]
```diff
@@ -164,6 +164,8 @@ const FeatureInfoSection = React.createClass({
         const catalogItemName = (this.props.catalogItem && this.props.catalogItem.name) || '';
         const fullName = (catalogItemName ? (catalogItemName + ' - ') : '') + this.renderDataTitle();
         const reactInfo = getInfoAsReactComponent(this);
+        // Silence lint
+        const ExtraComponent = "";
```
Comment (kring) [inline lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx]: You can disable eslint warnings on a line-by-line basis.  Search our code for `eslint-disable-line` for an example.

File: lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx [hunk 2]
```diff
@@ -204,6 +206,9 @@ const FeatureInfoSection = React.createClass({
                                 {reactInfo.info}
                             </Otherwise>
                         </Choose>
+                        <For each="ExtraComponent" index="i" of={FeatureInfoSection.extraComponents}>
+                            <ExtraComponent key={i} viewState={this.props.viewState}/>
```
Comment (kring) [inline lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx]: Is just the `viewState` enough context?  It seems like most of the section's props would be useful for the extra component, too.  Especially `feature` and `catalogItem`.

**Comment by kring** [comment]
Just those two comments.

**Comment by rdengate** [comment]
Thanks @kring! I have addressed both comments.

**Pull request merged**